static/locales/de.json

```diff
@@ -701,36 +768,56 @@
   "Kangaskhan": "Kangama",
   "Karate Chop": "Karateschlag",
   "Karrablast": "Laukaps",
+  "Kartana": "Katagami",
   "Kecleon": "Kecleon",
   "Keldeo": "Keldeo",
   "Kinesis": "Psykraft",
-  "King's Shield": "Königsschild",
   "Kingdra": "Seedraking",
   "Kingler": "Kingler",
+  "Kings Rock" : "King-Stein",
+  "King's Shield": "Königsschild",
   "Kirlia": "Kirlia",
   "Klang": "Kliklak",
   "Klefki": "Clavion",
   "Klink": "Klikk",
   "Klinklang": "Klikdiklak",
   "Knock Off": "Abschlag",
   "Koffing": "Smogon",
+  "Komala": "Koalelu",
+  "Kommo-o": "Grandiras",
   "Krabby": "Krabby",
   "Kricketot": "Zirpurze",
   "Kricketune": "Zirpeise",
   "Krokorok": "Rokkaiman",
   "Krookodile": "Rabigator",
+  "Kubfu": "Dakuma",
   "Kyogre": "Kyogre",
   "Kyurem": "Kyurem",
   "Lairon": "Stollrak",
   "Lampent": "Laternecto",
-  "Land's Wrath": "Bodengewalt",
   "Landorus": "Demeteros",
+  "Land's Wrath": "Bodengewalt",
   "Lanturn": "Lanturn",
   "Lapras": "Lapras",
+  "Large" : "Groß",
   "Larvesta": "Ignivor",
   "Larvitar": "Larvitar",
   "Laser Focus": "Konzentration",
+  "Last 12 hours": "den letzten 12 Stunden",
```

Comment:
sugg -> "Letzte 12 Stunden". Capital.

Reply:
See https://github.com/cecpk/RocketMAD/blob/master/templates/pokemon-history.html#L167 

Gives very bad dropdown. :/ And beginning of any option is capital. Also in German. ;) 
Where is it in a sentence?

Reply:
Pokemon History -> sentence is two parts (see comments)